README.md

```diff
@@ -1,119 +1,85 @@
-  -global-view string
-        Infoblox Network View for Global Address Space (default "default")
-  -grid-host string
-        IP of Infoblox Grid Host (default "192.168.124.200")
-  -local-network-container string
-        Subnets will be allocated from this container when --subnet is not specified during network creation (default "192.168.0.0/16")
-  -local-prefix-length uint
-        The default CIDR prefix length when allocating a local subnet. (default 24)
-  -local-view string
-        Infoblox Network View for Local Address Space (default "default")
-  -plugin-dir string
-        Docker plugin directory where driver socket is created (default "/run/docker/plugins")
-  -ssl-verify string
-        Specifies whether (true/false) to verify server certificate. If a file path is specified, it is assumed to be a certificate file and will be used to verify server certificate.
-  -wapi-password string
-        Infoblox WAPI Password
-  -wapi-port string
-        Infoblox WAPI Port. (default "443")
-  -wapi-username string
-        Infoblox WAPI Username
-  -wapi-version string
-        Infoblox WAPI Version. (default "2.0")
-  -http-request_timeout
-        Infoblox WAPI request timeout in seconds. (default 60)
-  -http-pool-connections
-        Infoblox WAPI request connection pool size. (default 10)
+### 1) Create configuration file for the plugin.
+create a file **`/etc/infoblox/docker-infoblox.conf`** and add the configuation parameters for the ipam-plugin. The configuration parameters are:
+
+| Option | Type  | Description |
+| ------ | ----- | ----------- |
+| grid-host string   | String | IP of Infoblox Grid Host
+| wapi-port  | String | Infoblox WAPI Port. (default "443")
+| wapi-username | String | Infoblox WAPI Username
+| wapi-password | String | Infoblox WAPI Password
+| wapi-version | String | Infoblox WAPI Version. (default "2.0")
+| ssl-verify  | String | Specifies whether (true/false) to verify server certificate. If a file path is specified, it is assumed to be a certificate file and will be used to verify server certificate.
+| http-request-timeout | Integer | Infoblox WAPI request timeout in seconds. (default 60)
+| http-pool-connections | Integer | Infoblox WAPI request connection pool size. (default 10)
+| global-view  | String | Infoblox Network View for Global Address Space (default "default")
+| global-network-container | String | Subnets will be allocated from this container when --subnet is not specified during network creation
+| global-prefix-length | Integer | The default CIDR prefix length when allocating a global subnet. (default 24)
+| local-view | String | Infoblox Network View for Local Address Space (default "default")
+| local-network-container | String | Subnets will be allocated from this container when --subnet is not specified during network creation
+| local-prefix-length | Integer | The default CIDR prefix length when allocating a local subnet. (default 24)
```

Comment:
For the descriptions, remove the periods at the end except where it is a complete sentence (for example, keep them for `ssl-verify` but remove them for `wabi-version` and `wabi-port`).